Common/Sources/Helper/Theme.swift

```diff
@@ -98,14 +119,16 @@ public class Theme: NSObject {
         public static let headerHeight = CGFloat(16)
         /// The divider UITableViewViewCell height.
         public static let dividerHeight = CGFloat(8)
+        /// The `UITableViewViewCell` tint color
```

Comment:
I know this is existing code but is `UITableViewViewCell` the correct name? View appears twice which looks odd